src/admin-views/settings/featured/link.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Help link for featured settings box.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views    $this  Template object.
```

Comment:
Here it's not an instance of `Tribe__Tickets__Admin__Views` it's an instance of `Tribe__Template`